code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -308,4 +308,79 @@
 	mag_type = /obj/item/ammo_box/magazine/tommygunm45
 	fire_sound = 'sound/weapons/Gunshot_smg.ogg'
 	can_suppress = 0
-	burst_size = 4
\ No newline at end of file
+	burst_size = 4
+
+/obj/item/weapon/gun/projectile/automatic/l85a2

```

Comment:
We honestly don't need more buffed weapons like this; certainly not ones which only science can produce. This is a dumb unbalanced addition.